bundles/org.openhab.binding.foobot/README.md

```diff
@@ -25,7 +25,7 @@ Bridge has the following configuration parameters:
 
 | Parameter        | Description                                           | Required
 |------------------|-------------------------------------------------------|----------
-| apikey           | API Key from https://api.foobot.io/apidoc/index.html  | Mandatory
+| apikey           | API Key from <https://api.foobot.io/apidoc/index.html>  | Mandatory
```

Comment:
Can you reformat the table alignment after this change, i.e. add two spaces/dashes at the end for this column in the lines before and after?